packages/frontend/src/content/CreateNote.tsx

```diff
@@ -26,9 +26,7 @@ const CreateNote = (props: RouteComponentProps) => {
       setErrorMsg(`File can't be bigger than ${MAX_FILE_SIZE / 1000000} MB`);
       return;
     }
-
-    setIsLoading(true);
-
+    startTransition(async() => {
```

Comment:
Looks like prettier was not run on this commit, as precommit hooks are not set up - and the project doesn't have VSCode configuration.

Can you manually run prettier on all files?
```console
$ npx prettier packages/**/*.tsx
``` 

To ensure this happens automatically in future, run
```console
$ npx husky install
```